src/components/input-field/input-field.tsx

```diff
@@ -462,7 +473,37 @@ export class InputField {
             );
         }
 
-        if (trailingIcon) {
+        if (
+            !this.isInvalid() &&
+            this.showLink &&
+            ['email', 'tel', 'url'].includes(this.type)
+        ) {
+            let link: string;
+            const props: { target?: string } = {};
+            switch (this.type) {
+                case 'email':
+                    link = `mailto:${this.value}`;
+                    break;
+                case 'tel':
+                    link = `tel:${this.value}`;
+                    break;
+                default:
+                    link = `${this.value}`;
+                    props.target = '_blank';
+            }
```

Comment:
This can also be extracted to a spearate function :slightly_smiling_face: 
E.g. `getLink`

Reply:
⚡